README.md

```diff
@@ -57,16 +90,41 @@ You can also use `make binaries-all` to prepare `buildkitd.containerd_only` and
 
 #### Starting the buildkitd daemon:
 
+You need to run `buildkitd` as the root user on the host.
+
 ```bash
-buildkitd --debug --root /var/lib/buildkit
+$ sudo buildkitd
 ```
 
+To run `buildkitd` as a non-root user, see [`docs/rootless.md`](docs/rootless.md).
```

Comment:
I think it'd be nice to add the header back here, to make this easier to find and have it stand out more 🤔